### ISSUE TITLE
A simple transfer-learning extension of Hyperband

### ISSUE THREAD

**Issue by sile** [sile/readings#21]
URL
-----

http://metalearning.ml/2018/papers/metalearn2018_paper32.pdf

Abstract
----------

> Hyperband [1] has become a popular method to tune the hyperparameters (HPs) of
expensive machine learning models, whose performance depends on the amount
of resources allocated for training. While Hyperband is conceptually simple,
combining random search to a successive halving technique to reallocate resources
to the most promising HPs, it often outperforms standard Bayesian optimization
when solutions with moderate precision are sufficient. In this paper, we propose
a model-based extension of Hyperband, replacing the uniform random sampling
of HP candidates by an adaptive non-uniform sampling procedure. We show that
our extension not only improves the precision resolution of Hyperband but also
supports transfer learning, both, within a Hyperband run and across previous HP
tuning tasks. We apply the method to the problem of tuning the learning rate when
solving linear regression problems and to the optimization of the HPs of XGBoost
binary classifiers across different datasets, showing that we favorably compare with
recently proposed extensions of Hyperband.

Table of Contents
--------------------

- (1) Introduction
- (2) Background
  - Bayesian optimization (BO)
  - Adaptive Bayesian linear regression (ABLR)
  - Hyperband
- (3) Combining ABLR with Hyperband
- (4) Experiments
  - (4-1) Tuning the learning rate for the stochastic optimization of linear regression models
  - (4-2) Tuning XGBoost binary classifiers
  - (4-3) Discussion

BibTeX
------

```
@article{valkovsimple,
  title={A simple transfer-learning extension of Hyperband},
  author={Valkov, Lazar and Jenatton, Rodolphe and Winkelmolen, Fela and Archambeau, C{\'e}dric}
}
```

**Comment by sile** [comment]
> We wish to combine ABLR with Hyperband to leverage the attractive properties of the two methods. Next, we describe our approach and argue that it simpliﬁes the methodology developed in #18.
